frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -0,0 +1,158 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Row, Col } from 'reactstrap'
+
+import { getImageThumbnail } from '../../utils'
+
+// Submission types
+const VIDEO = 'VIDEO'
+const PHOTO = 'PHOTO'
+const OTHER = 'OTHER'
+
+const YOUTUBE_BASE_URL = 'https://www.youtube.com/watch?v='
+const VIMEO_BASE_URL = 'https://vimeo.com/'
+
+const PhotoThumbnail = styled.img`
+  height: auto;
+  max-height: 5em;
+  max-width: 100%;
+  min-width: 3em;
+  width: auto;
+`
+
+const renderSubmissionByType = submission => {
+  switch (submission.entryType) {
+    case PHOTO:
+      return (
+        <Fragment>
+          <PhotoThumbnail
+            alt={submission.title}
+            src={`//localhost:3000/static/uploads/${getImageThumbnail(
+              submission.path
+            )}`}
+          />
+          <dt>Dimensions</dt>
+          <dd>{submission.horizDimInch} in. × {submission.vertDimInch} in.</dd>
+          <dt>Media Type</dt>
+          <dd>{submission.mediaType}</dd>
+        </Fragment>
+      )
+    case VIDEO:
+      if (submission.provider === 'youtube') {
+        return (
+          <Fragment>
+            <dt>YouTube Video</dt>
+            <dd>
+              <a href={`${YOUTUBE_BASE_URL}${submission.videoId}`}>
+                {`${YOUTUBE_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      } else {
+        return (
+          <Fragment>
+            <dt>Vimeo Video</dt>
+            <dd>
+              <a href={`${VIMEO_BASE_URL}${submission.videoId}`}>
+                {`${VIMEO_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      }
+    case OTHER:
+      return (
+        <Fragment>
+          <dt>Other</dt>
+          <dd>
+            <a href={submission.path}>
```

Comment:
I don't think this works, needs the base URL of the backend prepended (`//localhost:3000/...`)

Reply:
Are we trying to show both pictures and pdfs in a new tab here? We should at least be using something like this: ```            href={`//localhost:3000/static/uploads/${path}`}
            target='_blank'```